DevAcademy/Model/Properties.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+struct Properties {
+    var ogcFid: Int
+    var obrId1: URL
+    var druh: PossibleKind
+    var nazev: String
```

Comment:
az ted me to trklo... tvl borci od kdy programujem v cestine? Ted se nam to tam zacina misit, ze nektery promeny jsou v anglictine, nektery v cestine a vypada to jak nekde na cviku na vejsce. Pojdme to hodit do anglictiny at to nejak vypada a pristupujme k tomu kodu jako by to byl nas projekt. cc @mikolasstuchlik

Reply:
a nebo ted uz na to prcat. Upravim to v ramci lekce na API, kde ty modely budeme stejne nejak upravovat tem modelum co budou chodit z APIcka (nebo na tom aspon vysvetlim custom Codable).